tests/foreman/installer/test_installer.py

```diff
@@ -1353,6 +1378,19 @@ def extract_params(lst):
                 yield token.replace(',', '')
 
 
+def extract_sections(lst):
```

Comment:
No need to create a separate function to extract sections, we can generalize the existing extract params function so it will work for all the params(params, sections, subsections).

Reply:
Written/Modified `extract_params(lst)` to `extract_help(what)`.